ospray_modules/CMakeLists.txt

```diff
@@ -24,7 +24,6 @@ if(BRAYNS_OPTIX_ENABLED)
 endif()
 
 # OpenDeck module
-option(BRAYNS_OPENDECK_ENABLED "Activate OpenDeck module" OFF)
```

Comment:
revert. should work when the option stays here also for the plugin.

Reply:
Done